sig-policy/README.md

```diff
@@ -0,0 +1,20 @@
+# SIG Policy
+
+SIG Policy is responsible for defining, maintaining, and evolving network policy and security policy capabilities in Cilium.
+
+The [charter](charter.md) defines the scope and governance.
+
+## Meetings and Channel
+
+Join our [Slack channel](https://cilium.slack.com/archives/CDLN7836J) in the Cilium Slack, or join our meetings:  
+* Monthly on the second Tuesday at 11:00 - 12:00 US/Eastern (17:00 EU / 08:00 US-West)
+ - [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/94432191365?password=e89882f3-3eab-4371-aca6-8fd9d9eb1463)
+ - [Meeting Notes](https://docs.google.com/document/d/1p6LuzoKR55_HgQJTWFInwj2FlIeo7_uAFXYuXe_Lw7Q/edit?usp=sharing)
+
+## Leadership
+
+### Leads
+
+The Leads are responsible for driving the operations and community initiatives of SIG Community, as well as upholding the [community values](https://github.com/cilium/community/blob/main/VALUES.md).  
```

Comment:
```suggestion
The Leads are responsible for driving the operations and community initiatives of SIG Policy, as well as upholding the [community values](https://github.com/cilium/community/blob/main/VALUES.md).  
```

I think this needs wording to reflect driving initiatives in line with the charter, rather than operations and community initiatives?